### ISSUE TITLE
visualizing ground truth

### ISSUE THREAD

**Issue by VeeranjaneyuluToka** [mit-han-lab/bevfusion#442]
Hi,

I am trying to visualize the ground truth using visualize.py but GT is shifted. I have referred a couple of issues on the same and have done changes but still that did not help.

**My inputs:**
LiDAR point cloud is in FLU coordinate system.
GT is in ENU
yaw angle passed = -yaw - np.pi/2
origin is (0.5, 0.5, 0) but i have tried with (0.5, 0.5, 0.5) also,  still that did not help.

The result is as below
![image](https://github.com/mit-han-lab/bevfusion/assets/61545398/b6888d91-616f-44f2-9ae6-b586f8cdb43a)
It suppose to be blue bbox but it is shifted and drawn in red.

when i transform center (x, y, z) of GT from ENU to FLU (as my point clouds are in FLU and GT is in ENU) here line no 86(https://github.com/mit-han-lab/bevfusion/blob/main/mmdet3d/core/bbox/structures/lidar_box3d.py), GT gets drawn in right place as shown below
![image](https://github.com/mit-han-lab/bevfusion/assets/61545398/e9775b54-77fc-4f03-b770-922e7f89f08e)

Basically it is  in corners computation function and it is as below

1. Create corners in ENU coordinate system based on width, length, and height of bbox
2. Rotate using -yaw-np.pi/2 which means these corners are getting transformed to FLU from ENU
3. And then translate to center
In third step, (x,y,z) which is a center is still in ENU, that also needs to be in FLU, so when i do that change, it works fine.

If my understanding is correct, this change is required to visualize the GT or predictions irrespective of origin shift, is not it so or anything wrong in my inputs?

**Comment by Jayden9912** [comment]
Hi. I am solving the same issue. Did you found anything?
From the conversation in other issue, I also found that you are having problem training the camera only model, right?

**Comment by zhijian-liu** [comment]
Thank you for your interest in our project. This repository is no longer actively maintained, so we will be closing this issue. Please refer to the amazing implementation at [MMDetection3D](https://github.com/open-mmlab/mmdetection3d/tree/main/projects/BEVFusion). Thank you again!
